README.md

```diff
@@ -5,69 +5,11 @@
-
-Install the Cortex agent (an unmodified OSS Prometheus) on your Kubernetes cluster using the following command:
-
-    $ kubectl apply -n kube-system -f "https://cloud.weave.works/k8s/cortex.yaml?service-token=<token>&k8s-version=$(kubectl version | base64 | tr -d '\n')"
-
-Click Monitor to access the Cortex service in the Weave Cloud UI:
-
-<p align="center"><img src="imgs/monitor-cortex.png" alt="Prometheus Monitoring with Weave Cortex"></p>
-
-Weave Cloud gives you some example queries that displays general system information.  You can of course [write your own Prometheus queries](https://prometheus.io/docs/querying/basics/).
-
-
-## About Weave Cortex
-
-To learn more about Weave Cortex, consult the follow documents / talks:
-
-- [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
-- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
-- KubeCon Prometheus Day talk "Weave Cortex: Multi-tenant, horizontally scalable Prometheus as a Service" [slides](http://www.slideshare.net/weaveworks/weave-cortex-multitenant-horizontally-scalable-prometheus-as-a-service)
-- Or join us on [our Slack channel](https://slack.weave.works) and ask us directly!
-
-## Using Cortex on Other Platforms
-
-Cortex can be used to monitor virtually any application on any platform, as the local agent is a specially configured unmodified OSS Prometheus binary.  You can [download Prometheus from its website](https://prometheus.io/download/).  You will need v1.2.1 or later.
-
-When you've got Prometheus, you will need to [configure it to discover your services](https://prometheus.io/docs/operating/configuration/) and configure it to send its data to Weave Cloud by adding the following top-level stanza to `prometheus.yml`:
-
-    remote_write:
-    - url: https://cloud.weave.works/api/prom/push
-      basic_auth:
-        password: <token>
-
-Where `<token>` is the Service Token you obtained from Weave Cloud.
-
-Once your local Prometheus is running you can enter Prometheus queries into
-Weave Cloud. Go to https://cloud.weave.works and click Monitor from Weave Cloud header:
-
-<p align="center"><img src="imgs/monitor-cortex.png" alt="Prometheus Monitoring with Weave Cortex"></p>
-
-## Using Cortex to power Grafana dashboards
-
-To use the Prometheus Service with Grafana, configure your Grafana instance to have a datasource with the following parameters:
-
-- Type: Prometheus
-- Url: https://cloud.weave.works/api/prom
-- Access: Proxy
-- Http Auth: Basic Auth
-- User: `user` (this is ignored, but must be non-empty)
-- Password: <Service Token> (the same one your retrieval agent uses)
+If you would like to use Cortex, we recommend signing up for [Weave Cloud](https://cloud.weave.works) and following the instructions there.
```

Comment:
To use Cortex, sign up for [Weave Cloud](https://cloud.weave.works) and follow the instructions there.